compiler+runtime/src/cpp/jank/runtime/obj/range.cpp

```diff
@@ -234,6 +234,11 @@ namespace jank::runtime
     return runtime::to_string(seq());
   }
 
+  native_persistent_string obj::range::to_code_string()
+  {
+    return to_string();
```

Comment:
I would still do `return runtime::to_code_string(seq());` here, even though we know all `range` elements are numbers. If we end up adding different `to_code_string` implementations for numbers, we want that to reflect when use ranges of those numbers.